openlibrary/macros/SearchResultsWork.html

```diff
@@ -53,10 +53,11 @@ <h3 class="booktitle">
         <div class="decorations">
           $:decorations
         </div>
-      $if cta:
-        <div class="searchResultItemCTA-lending">
+
+      <div class="searchResultItemCTA-lending">
```

Comment:
availability.js relies on there existing a `.searchResultItemCTA-lending` somewhere.
`$cta=False` means we don't want to render CTA buttons for availability, e.g. in the case where we prefer for availability.js to render them. If `$cta=True` and `availability.js` is activated, we would get (and have been getting) 2 buttons.